lambdas/tests/unit/handlers/access-token-handler.test.ts

```diff
@@ -131,6 +139,109 @@ describe("access-token-handler.ts", () => {
                 expect(output.statusCode).toBe(200);
                 expect(metricsSpy).toHaveBeenCalledWith("accesstoken", MetricUnits.Count, 1);
             });
+
+            it("should return http 200 if the authorizationCodeExpiryDate is within date", async () => {
```

Comment:
We already have a 200 success test, I would rather see a 403 error test instead the whole point of this ticket was to try and increase test coverage for access-token-handler. Also I think you should mock or spy of the date, rather than making a call to getDate, the test could start feel under some circumstances

Reply:
I added a 403 test under the "fail paths" alongside a 200 test in the success paths.

Reply:
Please check the ticket details to see what I mean, we can have quick chat if it makes things clearer

Reply:
Updated `session-service#getSessionByAuthorizationCode` to check against current epoch time.
As discussed the 403 unit test is fine but I have also added a test to check the 403 when we have multiple session items.